lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -153,14 +153,22 @@ class TurboGraft.Remote
       initiator: @initiator
       xhr: xhr
 
-    if @refreshOnError
-      Page.refresh
-        response: xhr
-        onlyKeys: @refreshOnError
-    else if @refreshOnErrorExcept
-      Page.refresh
-        response: xhr
-        exceptKeys: @refreshOnErrorExcept
-    else
+    if TurboGraft.hasTGAttribute(@initiator, 'tg-remote-norefresh')
```

Comment:
I don't think `turbograft:remote:success:unhandled` needs to be an event as a 200 going unhandled isn't really a problem whereas a failing case demands a need to be handled.

I just realized that if `norefresh` is used in conjunction with `refreshOnError`/`refreshOnErrorExcept` the `unhandled` event will still be triggered. I'm going to update the readme to reflect this as it currently has no mention of `norefresh`.